CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -1,3 +1,22 @@
+# v1.13.2 - Changelog since v1.13.1
+
+## Changes by Kind
+
+### Bug
+
+- Fix error when no compute endpoint is passed ([#1622](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1622), [@pwschuurman](https://github.com/Sneha-at))
```

Comment:
This should be `Sneha-at` in the square brackets

Reply:
Thanks! Done